jakarta.ls/src/main/resources/transactions.json

```diff
@@ -1,3 +1,14 @@
 {
-
-}
\ No newline at end of file
+    "@Transactional": {
+      "prefix": "@Transactional()",
+      "body": [
+        "@Transactional(",
+        "\trollbackOn={SQLException.class, IOException.class},",
+        "\tdontRollbackOn={IllegalStateException.class})"
```

Comment:
Do we want to have a default rollback for all SQLException? Or do we want to add SQLWarning.class to the `dontRollbackOn` parameter, like in the example in the [specs](https://jakarta.ee/specifications/transactions/2.0/jakarta-transactions-spec-2.0.html)?

Reply:
The example of SQLException and SQLWarning specified in the [specs](https://jakarta.ee/specifications/transactions/2.0/jakarta-transactions-spec-2.0.html#:~:text=The%20following%20will%20cause%20the%20transaction%20to%20be%20marked%20for%20rollback%20for%20all%20runtime%20exceptions%20and%20all%20SQLException%20types%20except%20for%20SQLWarning.) is to illustrate the rule that the dontRollbackOn element will take precedence if both elements are specified. And the snippet here is to provide a usage example of rollbackOn and dontRollbackOn elements, and also provide some common used exception classes for developer. It may be better to keep the original exception classes.

Reply:
That's fair! I think it looks good.